sites/partners/pages/listings/[id]/applications/pending/index.tsx

```diff
@@ -113,6 +114,12 @@ const ApplicationsList = () => {
     formatLinkCell: isListingOpen ? formatDisabledCell : formatEnabledCell,
   }
 
+  const dayjsDate = dayjs(listingDto?.afsLastRunAt)
```

Comment:
looks like `listingDto.afsLastRun` will always exist, but if it doesn't this will return date and time of now. (same as if you just did `dayjs()`. Since it always exists it shouldn't be an issue, but it does make the check on line 204 unnecessary. In other words afsLastRun will always have some value so we don't need to do the check on 204

Reply:
Hey @ludtkemorgan I think this is here mostly because listingDto doesn't exist on the very first render cycle so without this it will throw an error as its looking for afsLastRunAt on an undefined variable 

I can pull the check on 204 though if you feel like its necessary though

Reply:
I guess it's better to be safe and just keep 204 check in. I don't think it's necessary, but it doesn't hurt to have that extra protection